easybuild/easyconfigs/f/fineRADstructure/fineRADstructure-20210514-foss-2023a.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'fineRADstructure'
+version = '20210514'
+
+homepage = 'http://cichlid.gurdon.cam.ac.uk/fineRADstructure.html'
+description = "A package for population structure inference from RAD-seq data"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/millanek/fineRADstructure/archive/']
+sources = [{'download_filename': '5896b9e.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['4f22232bb79dd7b589a1c8a389d7cca23372ea52f55205392ce9ce8f80aea5b2']
+
+dependencies = [
+    ('GSL', '2.7'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/finestructure', 'bin/RADpainter'],
+    'dirs': [],
+}
```

lets add some sanity check commands as well:
```suggestion
_bins = ['finestructure', 'RADpainter']
sanity_check_paths = {
    'files': ['bin/%s' % x for x in _bins],
    'dirs': [],
}

sanity_check_commands = ["%s -h" % x for x in _bins]
```